### PR TITLE
chore: update common.ts

### PR DESCRIPTION
**Pull request by eltociear**
addreses -> address

### CODE REVIEW

File: packages/core/types/src/customer/common.ts [hunk 1]
```diff
@@ -103,7 +103,7 @@ export interface CustomerAddressDTO {
 export interface FilterableCustomerAddressProps
   extends BaseFilterable<FilterableCustomerAddressProps> {
   /**
-   * Searches for addreses by properties such as name and street using this search term.
+   * Searches for address by properties such as name and street using this search term.
```
Comment (shahednasser) [inline packages/core/types/src/customer/common.ts]:
```suggestion
   * Searches for addresses by properties such as name and street using this search term.
```

**Review by shahednasser — CHANGES_REQUESTED** [review]
Nice catch! But I think it should still be plural.

File: packages/core/types/src/customer/common.ts (hunk 1 above)
Comment (eltociear) [reply packages/core/types/src/customer/common.ts]: Thanks!

**Review by shahednasser — APPROVED** [review]
LGTM thanks!

**Pull request merged**